src/lib/converter/converter.ts

```diff
@@ -256,9 +264,14 @@ export class Converter extends ChildableComponent<
      * @param context  The context object describing the current state the converter is in.
      * @returns An array containing all errors generated by the TypeScript compiler.
      */
-    private compile(entryPoints: readonly string[], context: Context) {
+    private compile(
+        entryPoints: readonly string[],
+        context: Context,
+        packages: PackageProgram[]
+    ) {
         const baseDir = getCommonDirectory(entryPoints);
         const entries: {
```

Comment:
We do manage to unify the different sources of program / entry point in to the `entries` here.